client/src/components/Payments/PaymentsPage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+
+import CardForm from './CardForm';
+
+const PaymentsPage = (): JSX.Element => {
+  const makeStripePromise = async () => {
+    const response = await fetch('/payments/public-key');
+    const { public_key: publicKey } = await response.json();
+    return await loadStripe(publicKey.toString());
```

Comment:
Make sure to call `loadStripe` outside of a component’s render to avoid recreating the `Stripe` object on every render